Release-Notes/Release3.md

```diff
@@ -172,6 +176,9 @@ updates for R3 at the end of April 2023.
 
 ## Future directions (selected Highlights)
```

Comment:
Am not really sure this belongs here. Where is the added value of this section? Once we publish the blog post it is there - before that no reason to talk about it.

Reply:
OK, then remove this section completely.